source/fe/mapping_q_generic.cc

```diff
@@ -3621,82 +3623,70 @@ namespace
    * Boundary<dim,spacedim>.
    */
   template <int dim, int spacedim, class TriaIterator>
-  void get_intermediate_points_on_object(const Manifold<dim, spacedim> &manifold,
-                                         const QGaussLobatto<1>        &line_support_points,
-                                         const TriaIterator &iter,
-                                         std::vector<Point<spacedim> > &points)
+  void add_intermediate_points(const Manifold<dim, spacedim> &manifold,
```

Comment:
I think the name of the original function was a mistake -- maybe call it `add_interior_points()`?